frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -233,6 +233,10 @@ function MainMenu(props: Props): ReactElement {
       coreMenuOptions.settings,
       coreMenuOptions.about,
     ]
+    // Insert an extra menu item if Static Embedded Apps are enabled
+    if (props.sharingEnabled) {
+      menuOptions.splice(3, 0, coreMenuOptions.saveSnapshot)
+    }
```

Comment:
Looks good and makes sense, just surprising to see in this PR. Not sure if this was something intentional

Reply:
Although it looks like the demo apps for the docs have `sharingEnabled`. Not sure if this is desired?

Reply:
> Looks good and makes sense, just surprising to see in this PR. Not sure if this was something intentional

Yup, intentional. Emiliano's work to integrate the S4A menu into core previously disabled this entirely, but it should still come up if the user sets the correct config options.

> Although it looks like the demo apps for the docs have sharingEnabled. Not sure if this is desired?

Oh huh, never noticed. I think it's fine here, since 1) it's disabled, 2) all the static embedded apps already have this issue, and 3) readers of the docs are rarely going to actually click into the app to see the menu.